sim_instr/src/sim_instr_node

```diff
@@ -0,0 +1,50 @@
+#include "ros/ros.h"
+#include "geometry_msgs/Twist.h"
+//#include <node_handle.h>
+#include <vector>
+#include <string>
+#include <fstream>
+#include <sstream>
+
+struct ParsedLine{
+    double time;
+    geometry_msgs::Twist twist;
+};
+
+ParsedLine parse_line(const std::string &line_in) {
```

Comment:
refactor this to be a function:
`std::istream& operator>>(std::istream &is, ParsedLine &line);`